src/Castle.Core.Tests/ModuleScopeTestCase.cs

```diff
@@ -149,11 +150,14 @@ public void SaveSigned()
 			CheckSignedSavedAssembly(path);
 			File.Delete(path);
 		}
+#endif
 
+#if FEATURE_ASSEMBLYBUILDER_SAVE
 		[Test]
 #if __MonoCS__
 		[Ignore("Expected: CastleDynProxy2.dll  But was:  /home/teamcity/buildagent/work/...")]
 #endif
+

```

Comment:
Any reason for inserting a new line here? Those attributes belong with that method.